code/src/index.css

```diff
@@ -1,13 +1,237 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	font-family: Helvetica, "Helvetica Neue", "Segoe UI", "Roboto", "Oxygen",
+		"Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", -apple-system,
+		BlinkMacSystemFont, sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	background-color: #333;
+	display: flex;
+	justify-content: center;
+}
+
+* {
+	box-sizing: border-box;
+}
+
+.header {
+	height: 80px;
+	display: flex;
+	align-items: center;
+}
+
+.main-content {
+	display: flex;
+	justify-content: space-evenly;
+	flex-direction: column;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+		monospace;
+}
+
+.app-header {
+	color: #fff;
+	font-size: 18px;
+}
+
+.section-header {
+	color: #fff;
+	font-size: 16px;
+	border-bottom: 1px solid #a0a0a0;
+	padding-bottom: 5px;
+}
+.app-container {
+	font-size: 14px;
+	display: grid;
+	grid-template-columns: repeat(1, 1fr);
+	grid-auto-rows: auto;
+	grid-gap: 1rem;
+}
+
+.album-container {
+	width: 300px;
+	position: relative;
+}
+
+a {
+	text-decoration: none;
+}
+
+.album-image {
+	width: 300px;
+}
+
+.album-name {
+	max-height: 15px;
+	color: #ffffff;
+	font-weight: bold;
+	display: block;
+	margin-top: 10px;
+}
+
+.album-link:hover > .album-image {
+	opacity: 0.5;
+}
+
+.album-link:hover > .action-bar {
+	opacity: 1;
+}
+
+.action-bar:hover {
+	opacity: 1;
+}
+
+.artist-container {
+	display: flex;
+	margin-top: 10px;
+}
+
+.artist {
+	margin-right: 0px;
+	display: flex;
+	flex-direction: row;
+}
+
+.artist-name {
+	color: #a0a0a0;
+}
+
+.artist:nth-of-type(n + 2)::before {
+	content: ", ";
+	margin-right: 4px;
+	align-self: center;
+	color: #a0a0a0;
+}
+
+.artist:last-of-type::before {
+	content: "& ";
+	margin-left: 4px;
+	margin-right: 4px;
+	align-self: center;
+}
+
+.artist:first-of-type::before {
+	content: "";
+	margin: 0px;
+	align-self: center;
+}
+
+.album-name:hover,
+.artist-name:hover {
+	text-decoration: underline;
+	color: #ffffff;
+}
+
+.action-bar {
+	width: 100%;
+	height: 65px;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: space-evenly;
+	position: absolute;
+	top: 30%;
+	opacity: 0;
+}
+
+.play-button {
+	width: 50px;
+	transition: all 1.3s;
```

Comment:
This is going to sound super silly but it stresses me that this button takes such a "long" time transitioning 😅